cypress/support/testHelper.js

```diff
@@ -17,3 +17,25 @@ export const shouldContainStyles = (element, css, styling) => {
     expect(el).to.have.css(css, styling);
   });
 };
+
+export const checkElementStyles = (elementString, text, color, fontFamily) => {
```

Comment:
This helper is named generically but is checking very specific styles. Is it needed? It might be better to just write in the test the explicit assertions so it is clear what is happening.

Reply:
We started out without this function. CodeClimate duplication error caused us to refactor. I agree that refactoring these makes the tests less readable. I dislike that CodeClimate forces us to refactor in this way. I see the same lack of readability in the unit tests in other parts of the codebase.

Reply:
I think we should have a wider discussion about readability vs optimal reuse of code with regards to CodeClimate.

Reply:
I get your point on reuse of code with regards to CodeClimate, however I am unsure that the comparison to unit tests is an exact comparison: it is not expected for unit tests to be refactored, whereas it is very likely that these tests will get split out of this article spec as we work through milestones.

Reply:
Even when these tests get moved to another file, the point about duplication will still remain. CodeClimate is clever enough to look at the whole codebase, not just in single files, so I don't see this being addressed in a future milestone.

Reply:
OK. Yeah I agree that there is little value in it at the moment.

How about we leave it as-is right now, and then re-assess when we have Cypress files for all the individual components? Otherwise we'll have red `x`s on every PR and will have to wait on you or Bogdan to merge PRs.

Reply:
Fine.

I have changes to the utils but let's fix forward.